agent/__init__.py

```diff
@@ -642,6 +648,29 @@ def write_metadata(data, rewrite_file):
     metadata_path.chmod(0o644)
 
 
+def enroll_device(enroll_token, claim_token, device_id):
+
+    payload = {
+        'key': enroll_token,
+        'claim_token': claim_token,
+        'device_id': device_id
+    }
+
+    try:
+        enroll_req = requests.post(
+            '{}/v0.2/enroll-device'.format(WOTT_ENDPOINT),
+            json=payload
+        )
+    except requests.exceptions.RequestException:
+        logger.exception("enroll_device :: rises exception:")
+
+    if not enroll_req.ok:
+        logger.error('Failed to enroll device...')
+        req_error_log('post', 'Enroll by token', enroll_req, caller='enroll-device')
+    else:
+        logger.info(enroll_req.json()['message'])
```

Comment:
As I said in the task's API side PR, we don't need pass some text over internet to inform client about everything went well

Reply:
Fixed